ntbs-service/wwwroot/source/Components/FilteredDropdown.ts

```diff
@@ -164,6 +168,19 @@ const FilteredDropdown = Vue.extend({
 
             optionInnerHtml += "</optgroup>";
             return optionInnerHtml;
+        },
+        findSelectFieldInContainer(container: Vue) {
+            if (container.$refs["selectField"]) {
+                return container.$refs["selectField"];
+            }
+            let selectField: Vue;
+            for(let i = 0; i < container.$children.length; i++) {
+                selectField = this.findSelectFieldInContainer(container.$children[i]);
+                if (selectField) {
+                    return selectField;
+                }
+            }
+            return;
```

Comment:
It's probably better to return `null` rather than `undefined` here, though it won't make a functional difference